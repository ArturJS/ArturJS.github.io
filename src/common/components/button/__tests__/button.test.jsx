import React from 'react';
import render from 'preact-render-to-json';
import Button from '../button.jsx';

describe('<Button />', () => {
    it('renders correctly', () => {
        const tree = render(
            <Button type="submit" theme="hot" className="btn-test-class">
                <span>Inner content...</span>
            </Button>
        );
        expect(tree).toMatchSnapshot();
    });
});
