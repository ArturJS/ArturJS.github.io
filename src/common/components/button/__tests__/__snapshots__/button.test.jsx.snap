// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<button
  class="btn btn-gradient-basis btn-hot btn-test-class"
  type="submit"
>
  <span
    class="btn-content"
  >
    <span>
      Inner content...
    </span>
  </span>
</button>
`;
